Exam/CC1/src/exercice2/SortedContainer.java

```diff
@@ -0,0 +1,61 @@
+package exercice2;
+
+public class SortedContainer implements ISortable {
+
+	int[] elements = new int[15];
+	int tailleTableau =0;
+
+
+	@Override
+	public void sort(int[] list, int listSize) {
+		int longueur = listSize;
+		int tampon = 0;
+		boolean permut;
+
+		do {
+			permut = false;
+			for (int i = 0; i < longueur - 1; i++) {
+				if (elements[i] > elements[i + 1]) {
+					tampon = elements[i];
+					elements[i] = elements[i + 1];
+					elements[i + 1] = tampon;
+					permut = true;
+				}
+			}
+		} while (permut);
+
+	}
+
+	public SortedContainer(int[] list) {
+		this.tailleTableau = list.length;
+		
+		for(int i =0; i<15; i++) {
```

Comment:
Ok but carefull with *magic* numbers.
15 is actually `elements.length` so prefer this instead.